e2e/test/E2ETests.csproj

```diff
@@ -139,4 +139,4 @@
     </PackageReference>
   </ItemGroup>
 
-</Project>
\ No newline at end of file
+gi</Project>
```

Comment:
"gi"?

Reply:
Not sure how did that come here. Thanks for noticing. updated.